cmd/crowdsec-cli/utils.go

```diff
@@ -732,3 +733,18 @@ func formatNumber(num int) string {
 	res := math.Round(float64(num)/float64(goodUnit.value)*100) / 100
 	return fmt.Sprintf("%.2f%s", res, goodUnit.symbol)
 }
+
+func getDBClient() (*database.Client, error) {
```

Comment:
there is already two implementations of `getDBClient` : in pkg/apiserver and pkg/exprhelpers. Can you either either of those ?

Reply:
I am afraid I either I don't understand what you meant, or I miss something here.

`pkg/exprhelpers` db stuff seems to me completely unrelated as it seems to require something already initialized. I don't think there's any implementation of this in `pkg/apiserver` There's one implementation in `pkg/database` that is used here.

In fact this function check that there's a database then it returns a client to it.

Reply:
nevermind I read way too fast :<